# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/libtiff/jpeg-zlib.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./config.site.orig	Tue Jul 18 04:09:20 2000
+++ ./config.site	Tue Jul 18 04:11:18 2000
@@ -41,8 +41,8 @@
 # Package controls.
 #
 #DSO="auto"			# auto|IRIX|IRIX52 enable DSO support for system
-#JPEG="no"			# yes|no configure JPEG support (see below)
-#ZIP="no"			# yes|no configure Deflate support (see below)
+JPEG="yes"			# yes|no configure JPEG support (see below)
+ZIP="yes"			# yes|no configure Deflate support (see below)
 #LIBGL="auto"			# yes|no|auto configure IRIS GL-based tools
 #LIBIMAGE="auto"		# yes|no|auto configure SGI RGB image tools
 #HTML="no"			# yes|no install HTML documentation
@@ -96,6 +96,7 @@
 #
 #DIRS_LIBINC="$DIRS_LIBINC ../src/jpeg-6b" # directory for jpeg include files
 #DIR_JPEGLIB=../src/jpeg-6b		# directory for libjpeg
+DIR_JPEGLIB=/usr/lib		# directory for libjpeg
 
 #
 # Deflate-specific parameters; used when Deflate support is enabled (see above).
@@ -110,7 +111,7 @@
 #     Look for it also at ftp://ftp.uu.net/pub/archiving/zip/zlib.
 #
 #DIRS_LIBINC="$DIRS_LIBINC ../zlib"	# directory for zlib include files
-#DIR_GZLIB="../zlib"			# directory for libz
+DIR_GZLIB="/usr/lib"			# directory for libz
 
 #
 # Miscellaneous parameters.
